**Proposed Title2**: Application of Near-infrared spectroscopy with Artificial Neural Netwrok for rapide intact-seed analysis, genotype classification and discrimination in multi-environment using African Core-collection and interspecfic AB-QTL population in peanut (Arachis hypogaea L.)



**Abstract** 

Peanut is an important oilseed crop in the world, and the need to analyze genetic variation of seed quality in the cultivated and wild relatives is essential to develop cultivars with superior nutritional profiles. Current methods for analyzing seed quality are generally based on wet chemical analyses, which are time-consuming, complex and irreversible destructive making them not efficient for large-scale screening. In this study, we applied Near Infrared Reflectance Spectroscopy (NIRS) to analyze germplasm variability and discriminate and classify intact-seed of 798 samples, collected and assembled in four genetic and environment-based sets: one set of 300 African varieties of a core-collection and three sets of 133 interspecific genotypes of an AB-QTL population, field evaluated in a large spatial scale of 2 countries, comprising 3 environments, Mbalmayo and Bafia in Cameroon and Nioro in Senegal, at the 95-day stage of growth under rain-fed conditions in 2021. NIRS elemental spectra were gathered on six subsets of intact-seed of each of the 798 samples, after 3 rotation scans, with a spectral resolution of 16 cm-1 over the spectral range of 867-2530 nm. NIRS spectra were processed by principal component analysis (PCA) combined with deep learning, with the aim of obtaining a classifier able to yield the corresponding confusion matrices and so discriminate and classify the studied genotypes. As results, a huge significant variation was found between varieties and interspecific genotypes, within and between the 3 environments for all spectra wavelength. The magnitude of genetic variability was particularly observed for 11 major wavelength such as 1723 nm, usually related to oil content and fatty acid composition. The interspecific population covers remarkably half of the spectrum of the core-collection, with a positive variation among some interspecific genotypes that outperform African varieties. PCA analysis of spectra yielded the most quality attribute in the first PCA component with 67% of the variation, and the first three components retain most of the cumulative variation, i.e., 92%, needed for the both discrimination and classification of genotypes and environments into 4 clusters, corresponding to the 4 samples sets. The two most separated environments determined by all PCA components were Mbalmayo and Bafia. Interestingly, a deep learning predictive model was constructed and a correct classification of 95.4 % was achieved for the 4 studied sets, aiming to properly classify each genotype-specific spectrum, as well as, to identify genetic and environment origins of intact-seed peanut. The confusion matrix achieved for the two sets of Mbalmayo and Nioro shows 99% of instances classified correctly with 99% at both sensitivity and specificity, confirming that their intact-seed composition is very different from each other and from those of the other intact-seed samples. Overall, NIRS combined with machine learning can be used to accurately classify and identify genetic and environment origins of intact-seed peanut, as well as, for future nutritional and QTL analyses for various applications including genetic breeding in multi-environment, intra and interspecific context. 

**Key words:** Intact-seed, NIRS, Machine learning, Classification, Environment, Interspecific, nutritional, Breeding

**Introduction**

Peanut also known as groundnut, is an annual grain legume cultivated globally on 36.18 million hectares of area, yielding 71.68 million metric tons of pods in 2020 (FAOSTAT, 2020). As a functional food, peanut seed contains 34 to 56% oil, 22 to 30% protein, 10 to 25% carbohydrates, and 0.05 to 1% of various nutrients, beneﬁcial to human health such as vitamin E, K and B complex, folic acid, niacin, antioxidants, and biologically active polyphenolics, including ﬂavonoids and isoﬂavones and minerals (Ca, P, Mg, Zn, and Fe) (Harch et al., 1995; Janila et al., 2013; Desmae et al., 2019). The main production constraints of the crop include drought, pests, diseases, and environmental changes. The oil and protein content, amino and fatty acid composition of seeds and haulms, shelf life, aroma, flavor, cooking quality and cooking time are all affected by these constraints. There is an effort to improve peanut yield as well as seed composition related traits such as fatty acid composition. The presence of relative proportions of various fatty acids affects the nutritional quality, health, flavor and shelf-life of peanut seeds and products (Teres et al., 2008; Vassiliou et al., 2009; Carrillo et al., 2012; Parilli-Moser et al., 2022). For instance, high-oleic acid has nearly 10-times more auto-oxidative stability over linoleic acid, thus giving longer shelf life to peanuts products, contributing to human nutrition (Davis et al., 2016; Nawade et al., 2018; Desmae et al., 2019). Consequently, seed nutritional quality traits are targets for improvement through genetic breeding. 

According to genetic and molecular data, the cultivated peanut (*Arachis hypogaea* L.) is a self-pollinated, allotetraploid (AABB genome; 2n=4x= 40) with limited genetic variation, most probably the result of the crop's origin, from a single hybridization event between two wild diploid species *A. duranensis* (A genome) and *A. ipaensis* (B genome), followed by a rare and spontaneous chromosome doubling (Simpson et al., 2001; Burow et al., 2009) that complicates sharing of diversity between cultivated and wild diploid species. The narrow genetic base of cultivars coupled with low utilization of genetic resources are the major factors limiting peanut breeding and productivity globally. Exploitation of new and diverse sources of variation is needed to enlarge the genetic variability of peanut and valuable efforts are in progress. Hence, to resolve the issues of ploidy differences, synthetic compatible tetraploid have been developed (Favero et al., 2006; Mallikarjuna et al., 2011b) from diploid wild relatives that exhibited a high level of genetic variation (Barkley et al., 2007; Bechara et al., 2010) with potential trait variation that may useful in peanut breeding through interspecific cross. Likewise, a number of genetic approaches, such as Advance Backcross Quantitative Trait Loci (AB-QTL) and Chromosome Segment Substitution Lines (CSSL) are currently applied to enlarge the genetic base of cultigen and to map and facilitate wild introgression from wild relatives into elite lines (Fonceka et al., 2012a, 2012b; Tossim et al., 2020). Such wild polymorphisms have the potential for use in further improving peanut quality.

The improvement of quality traits largely depends on the magnitude of genetic variability. Although significant efforts have been devoted to characterizing cultivars and germplasm collections for simple traits and for the most important agronomic traits (yield and resistance to pests and diseases) (Upadhyaya, 2005; Upadhyaya et al., 2006, 2011; Mallikarjuna et al., 2011a; Kumari et al., 2014), less is known about other traits related to nutritional (Grosso et al., 2000), culinary, or sensory attributes (Bianchi-Hall et al., 1993). This is mainly due to the fact that these traits are quantitative and multigenic, with low heritability and strong genotype by environment interactions (GxE) (Grosso et al., 1994; Isleib et al., 2008), thus making it difficult to assign a trait value for each accession. Moreover, the phenotyping of these traits is expensive in terms of both direct monetary input and human labor, time‑consuming, complex, and irreversibly destructive. Another main factor generally limiting chemical studies are the difficulties to analyze many samples, each requiring many seeds. For instance, for average fatty acid composition measurements, the standard practice is to comminute 100+ grams of sample, express oil from this ground sample, and measure the average fatty acid composition of the expressed oil by gas chromatography (GC) (Nawade et al., 2018; Davis et al., 2021). Efforts to improve seed quality would be aided with rapid and nondestructive technologies. More rapid and less expensive approach to predict chemical content among samples have evolved, and these include modiﬁed refractive index, capacitance sensor (Kandala et al., 2008), hyperspectral imaging (Huang et al., 2014; Rabanera et al., 2021) and near infrared (NIR) (Govindarajan et al., 2009; Tao et al., 2019; Davis et al., 2021). Among these, NIR-based methods are the most rapid, make it possible to analyze large collections of samples and for peanut breeders, an added beneﬁt with NIR is the method can be nondestructive. Moreover, in recent years, many scholars have applied machine vision, deep learning, and spectral sensing technology to seed quality detection, and it has advanced research in plants including variety identification in maize (Wang and Song, 2023; Xu et al., 2023), protein detection in barley (Singh et al., 2023) or molecules classification in medicine (Shang et al., 2023; Tian et al., 2023).

In the current study, we applied NIRS combined with deep learning to analyze germplasm variability, discriminate and classify intact-seed of 798 samples. These samples were collected and assembled in four genetic- and environment-based sets: one set of 300 African varieties of a core-collection and three sets of 133 interspecific genotypes of an AB-QTL population, field evaluated in a large spatial scale of 2 countries, comprising 3 environments, **Mbalmayo** and **Bafia** in **Cameroon** and **Nioro **in **Senegal**. More precisely, the objectives were to i) realize a rapid NIR measurement to analyze range of genetic variation of intact-seed constituents based on NIR spectral profile of a large number of samples in a large spatial scale of 3 environments, ii) study the pattern of genetic variability of the interspecific population in comparison to the core-collection, iii) possibly discriminate genetically close interspecific genotypes within and between environments, and iv) develop a deep learning classification model and study its performance to classify and predict the genotype-specific spectra according to genetic and environmental origins. We used a large number of samples in a large spatial scale of 3 environments to examine how the environment factor can influence seed composition and discriminate genotypes. With these objectives, the study highlights huge and very subtle genetic and environmental variations of intact-seed constituents using NIRS spectra combined with multivariate analysis. This paper proposed a deep learning predictive model based on NIRS spectra with a robust capacity for accurate classificartion of the intact-seed peanut samples, as well as for future nutritional and QTL analyses for breeding applications in the multi-environment, intra and interspecific context.

**Materials and methods**

**Genetic and plant material** 

Two distinct genetic materials were used in this study: an interspecific advanced backcross population of 133 genotypes and a core collection of 300 African cultivars. The advanced backcross population of 133 BC2F4 derivatives was developed from the initial interspecific cross using Fleur11 as recurrent cultivated parent and the wild synthetic tetraploid ‘ISATGR 278-18’ (Nguepjop et al., 2016). The recurrent parent used, Fleur 11, is an elite Spanish-type variety, widely cultivated in West Africa. The wild parent ISATGR 278-18 is derived from a cross between *A. batizocoi* ICG 13160 (GKBSPSc 30082, PI 468328) and *A. duranensis* ICG 8138 (GKP 10038, PI 262133) (Mallikarjuna et al., 2011b). The African core collection of 300 cultivars was defined based on breeder’s knowledge and on diversity data from a collection of 1050 accessions (breeding lines and landraces) held by 10 breeding programs in East, Southern and West Africa, assembled thorough the Peanut Innovation Lab (PIL) initiative and the dP IAVAO, the West African research network devoted to plant breeding. This collection is shared and currently used by those 10 breeding programs for multivariate analysis related to adaptation, yield components, nitrogen fixation, and pest and disease resistance.

**Intact-Seed provenance, trials location and field experimental design**

A total of 798 intact-seed samples used in this study were collected from four field experiments. The field experiments were conducted in 3 different locations in 2 countries, Mbalmayo and Bafia in Cameroon and Nioro in Senegal, under rain-fed conditions in 2021. The three experimental locations were chosen to meet environmental differences from the other locations, based on different criteria, largely based on ecology (climate and vegetation), agro-ecological zones, tradition of peanut cultivation and economic consideration. Bafia is one of the main areas of peanut production in Cameroon. It is located in tropical savannas and have yellow vertisol soil (Temga et al., 2021) with a transitional equatorial climate of the Sudano-Guinean type with an average temperature of 25.1°C and annual average rainfall of 1500 mm. Mbalmayo is located in the tropical forest area of Cameroon and has ocher vertisol soil (Temga et al., 2021) with a bimodal humid-forest rainfall climate characterized by an average temperature of 26.5°C and rainfall of 2402.8 mm. Nioro is located in the South part of the Senegalese peanut basin with sahelian semi-humid ecology and has a Deck Dior soil, silty-sand fine sandy, mixed haplic ferric lixisol, a leached ferruginous tropical soil (Bogie et al., 2018). The mean annual precipitation is 758 mm and mainly comes between July and September with an average temperature of 30 °C. The fields at Bafia and Mbalmayo were one-year fallow land after maize cultivation by farmers and were acquired from the farmers and cleared and plowed for the purposes of the study. The cultural precedent at Nioro was millet. The experiments in Bafia and Mbalmayo were conducted during one of the two rainy seasons from April-July, while the Nioro experiment was done during the rainfall season, July-October at the Research Station of the Centre National de Recherche Agronomique (CNRA). The interaction between the environmental (climate, soil and vegetation) and spatial factors creates a very complex system of environments that affect/confront peanut plant growth and development.

The same experimental design and the common agricultural practices, from sowing to harvest, were used in each of the 3 environments. For each environment, an alpha-lattice experimental design was used with three replications and 10 blocks per replication. For the interspecific AB-QTL population and core collection, the blocks contained 14 and 10 elementary plots, respectively. A plot consisted of rows of 3 m on which 10 plants of the same genotype were sown with a spacing of 30 cm between plants on the same row and 50 cm between two adjacent rows. The seeds were treated with Granox (captafol 10%, benomyl 10%, and carbofuran 20%) before planting to protect them against parasitic attacks and one seed per hill was sown manually at 4 cm depth. According to usual peanut cultural practice, one hundred and fifty kg/ha of mineral fertilizer (6-20-10) were added 20 days after sowing. Throughout the vegetative development, weeds were manually harvested. The harvest was done at 95 days after sowing, followed by free-air drying for one month.

**Intact-seed sample preparation**

At the end of the pod-drying stage, pods of all plants of each genotype from each of the three agronomic replicates were separated from haulms, stored and dehulled before NIRS analyzes. Intact-seeds from pods of the three agronomic replicates were bulked into one specific sample, stored in biodegradable plastic bag, and labelled according to their respective genotype name and environment of origin. Thus, intact seeds of each sample for NIRS analysis came from pods of 25-30 harvested plants of each genotype. Finally, a total of 798 samples of intact-seed were assembled in four genetic- and environment-based sets: three sets of the133 advance backcross genotypes, each from one of the three studied environments, Bafia, Mbalmayo (Cameroon) and Nioro (Senegal), and one set from the 300 African cultivars from the Nioro. 

**NIRS spectra acquisition** 

NIRS spectra acquisition was performed to generate a reference spectra database for the present study and further analysis. Here, spectra acquired were used to analyze germplasm variability and to discriminate and classify samples. Prior to recording spectra, a reference spectralon were measured. NIRS elemental spectra were then acquired on six subsets of intact-seed of each of the 798 samples. The six subsets of each sample were used, as six replicates, to minimize uncertainties due to the hypothetical heterogeneity of intact-seed of samples. More precisely, intact seeds of each sample, contained in the plastic bags, were six-fold randomly sampled to provide biological and analytical replicates, from each other to cover the whole sample. Intact-seeds of each subset were then loaded in the ring cup with an internal diameter of 5.1 cm and the six subsets of each sample were measured in sequence. Spectra of each of the six subsets were gathered after 3 rotation scans with a spectral resolution of 16 cm-1 over the spectral range of 867.5–2530 nm (3952-11528.cm-1) wavelength, using Tango spectrometer from Bruker. Cup surface were cleaned to avoid interference between two consecutive samples. At the end, each sample was analyzed in six replicates, and the mean spectrum reading was used for analyses. 

**PCA Analysis**

PCA was applied to discriminate the varieties and the genetically related interspecific genotypes within and between environments based on spectra profile. PCA is a multivariate unsupervised statistical method able to project multivariate data and describe relevant trends in the analyzed dataset without affecting the relations between the samples. PCA can also reveal as variables that determine some inherent structure of the data, which can be interpreted in chemical terms. The reduction of the number of variables is achieved by making a linear combination of original variables, which yields the so-called principal components (PC) that are decorrelated with each other, i.e., the information contained in one is not present in the other and are defined so to describe the direction of maximum variance of the original data. PCA was conducted on the pretreated spectra using the R rchemo and rnirs packages. The PCA has been coupled with Mahalanobis distances to reduce dimensionality before carrying out the discriminant analysis. The Mahalanobis distances are determined in units of standard deviations from the center (mean) of the dataset. In this study, the following PCA information was considered (i) the scree-plot, to estimate the variance of data according to each PC; (ii) the score plot, to visualize the projection of the sample on each PC; and (iii) the loading plot, to evaluate the influence of each variable, i.e., absorbance peak areas, on each PC, i.e., the relationship between the original variables and the subspace dimension. Plots of PCs versus each other represents how the variables that they account for are related. Thus, PCA allows emphasizing and interpreting variables and all relevant differences among genotypes within and between environments. The broadband spectra were previously pre-treated to highlight the relative spectral variations, reduce NIRS signal uncertainties resulting from fluctuations of light energy and matrix effects, and enable spectra comparison. 

**Statistics** 

The OPUS spectroscopic software (version 8.1.29, Bruker) was used to visualize raw data and perform basic statistic. The absorbance of each wavelength was transformed into Log (1/R), where R represents reflectance, since this variable correlate better with chemical components than with raw reflectance. Principal components (PC) were used to create a multivariate discriminant model as described above. Pre-treatments were employed to correct the spectrums during the data analysis. The qualitative absorbance analysis was performed by using the characteristic wavelength peaks selected by applying the paired Student’s t-test with 5% of significance to each point of the spectrum by comparing pairs of investigated genotypes. P-values lower than 0.05 allowed finding the regions of the spectrum showing statistically significant differences among and between the interspecific derivatives and cultivars. The absorbance peaks of the wavelengths that mostly differed among the genotypes were identified. The spectral information was used when establishing the PCA classification over the spectral range selected from 1000 to 2500 nm. In order to reduce the data dimensionality and simplify the complexity of computation, we selected the optimal characteristic wavelengths, probably containing component-specific and sample-specific information for establishing the discrimination models instead of using the full spectra.

**Classification model using deep learning based on NIR spectra**

Machine learning was applied to classifying varieties and interspecific genotypes thorough modelling and prediction of genotype-specific spectra according to genetic and environmental origin. Artificial neural networks (ANNs), deep neural networks (DNNs), convolutional neural networks (CNNs) (Liu et al., 2023b, 2023a; Park, 2023; Shang et al., 2023; Singh et al., 2023), recurrent neural networks (RNNs) (Wang and Song, 2023) and long short-term memory (LSTM) networks (Tian et al., 2023) are some of the most fascinating machine learning models that have been iteratively applied to analyze nonlinear and complex spectra data of the study. Prior to applying deep learning algorithms, the raw NIR spectra were pre-processed to remove any noise and normalization. The pre-processed raw data have been split into training and testing. The training set is used to train the deep learning model, while the testing set is used to evaluate its performance. The choice of model depends on the complexity of the data and the task at hand. The deep learning model learns to recognize patterns in the data and optimize its parameters to minimize the loss function. In DL, metrics such as accuracy, precision, recall, and F1 score was used to assess the model's performance. To optimize the model, the CNN or ANN consist of adjusting the hyperparameters of the model to improve its performance such as the learning rate, batch size, and number of epochs. Once the model is trained and optimized, it can be used to predict the intact-seed variety or interspecific derivatives labels of new NIR spectra. In this work, after iterative analyses the DL ANN approach was applied on four peanut sets. The network consisted of 8 hidden layers with the succession regularization layers of drop-out and batch normalization. The drop-out rate was 30% (first layer) and 20% (second layer). For the optimizer, the Gradient Descent with the Root Mean Squared Propagation was used. All models and derivate results are shared on GitHub as supplementary results.

**Results** 

**Quality control of NIRS spectra** 

Quality control of spectra was performed to identify atypical spectra and to check variation among the six subsets of each of the 798 samples. As results, two distinct elemental spectra (0.04%), were identified as an outlier (supplementary Figure 1) and were considered as non-viable spectra and were discarded for further analyses. A total of 6 working-days were required for spectral acquisition of the 798 samples, each repeated 6 times and gathered after 3 rotation scans. We performed ACP analysis to check the effect of date on spectra acquisition and no cluster related to date was found (supplementary Figure 2), indicating that there were stable lab conditions during spectra acquisition. The Mahalanobis statistic was used to assess variation among the six subsets of each of the 798 samples. With few exceptions, the Mahalanobis distance between the six subsets of each sample was consistent among samples (Figure 1). For this reason, the original spectra graph is presented in Figures 3 as the average absorption of each sample from the six replicated spectra. Finally, although not all wavelength regions contain information about seed quality traits, these regions were not excluded for the deep learning application even as noisy parts. The 2 non-viable spectra were eliminated from both PCA and deep learning process since it was determined as an outlier.